tests/pages/MeetingWidget.page.js

```diff
@@ -3,12 +3,21 @@ class MeetingWidgetPage {
   get displayWidgetBtn() { return $('button[aria-label="Display Meeting Widget"]'); } // #md-button-3
   get interstitialMeeting() { return $('.wxc-interstitial-meeting'); }
   get meetingInfo() { return $('.wxc-meeting-info'); }
+  get joinMeetingBtn() { return $('.meeting-controls-container button[alt="Join meeting"]'); }
+  get waitingForOthers() { return $('h4=Waiting for others to join...'); }
 
   loadWidget(meetingDestination) {
     this.destination.setValue(meetingDestination);
     this.displayWidgetBtn.click();
     this.interstitialMeeting.waitForDisplayed({timeout: 30000});
   }
+
+  waitForWaitingForOthersScreen() {
+    browser.waitUntil(() => this.waitingForOthers, {
```

Comment:
`this.waitingForOthers` is always truthy. Please use https://webdriver.io/docs/api/element/waitForDisplayed/ like here: https://github.com/webex/widgets/pull/54/files#diff-063a925e621a59156c6d99c3a82c63ed56563e2235370893ee1c8699dbe6f542R12